drivers/memory_window/dynamic_slots.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _DYNAMIC_SLOTS_H_
+#define _DYNAMIC_SLOTS_H_
+
+#include <stdint.h>
+
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/devicetree.h>
+
+#include <mem_window.h>
+
+#define ADSP_PAGE_SIZE 0x1000
+#define MEMORY_WINDOW_SLOT_SIZE     ADSP_PAGE_SIZE
+#define MEMORY_WINDOW_SLOT_INVALID_RESOURCE_ID    0xffffffff
+#define MEMORY_WINDOW_SLOTS_COUNT   7
+
+#define ADSP_SUCCESS 0
+#define ADSP_ERROR -1
+#define ADSP_NOT_FOUND 9
+#define ADSP_MM_L2HP_CANNOT_MIRROR 588
+
+#pragma pack(push, 4)
+enum SlotTypes {
```

Comment:
I need the Zephyr experts to weigh in on, but I haven't seen much CamelCase in Zephyr codebase.

Reply:
No need to. My mistake. Already fixed.